standard/bridge-v1/pkg/util/util.go

```diff
@@ -0,0 +1,49 @@
+package util
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"strings"
+)
+
+// NewLogger initializes a *slog.Logger with specified level, format, and sink.
+//   - lvl: string representation of slog.Level
+//   - logFmt: format of the log output: "text", "json", "none" defaults to "json"
+//   - tags: comma-separated list of <name:value> pairs that will be inserted into each log line
+//   - sink: destination for log output (e.g., os.Stdout, file)
+//
+// Returns a configured *slog.Logger on success or nil on failure.
+func NewLogger(lvl, logFmt, tags string, sink io.Writer) (*slog.Logger, error) {
+	level := new(slog.LevelVar)
+	if err := level.UnmarshalText([]byte(lvl)); err != nil {
+		return nil, fmt.Errorf("invalid log level: %w", err)
+	}
+
+	var (
+		handler slog.Handler
+		options = &slog.HandlerOptions{
+			AddSource: true,
+			Level:     level,
+		}
+	)
+	switch logFmt {
+	case "text":
+		handler = slog.NewTextHandler(sink, options)
+	case "json", "none":
+		handler = slog.NewJSONHandler(sink, options)
+	default:
+		return nil, fmt.Errorf("invalid log format: %s", logFmt)
+	}
+
+	var args []any
+	for i, p := range strings.Split(tags, ",") {
```

Comment:
This for-loop errors out if `tags` is an empty string (as is the case by some calling code). Simple check for
```go
if tags != "" {
     for i, p := ......
```
Seemed to fix the issue